net/src/main/java/io/bitcoinsv/jcl/net/protocol/handlers/blacklist/BlacklistHandlerImpl.java

```diff
@@ -177,30 +220,37 @@ private synchronized void updateState(PeersBlacklistedEvent.BlacklistReason newR
      * It blacklists the Host given for the reason specified.
      * NOTE: The "duration" specified the duration of the blacklist, overriding the one defined in the reason itself.
      */
-    private void blacklist(BlacklistHostInfo hostInfo, PeersBlacklistedEvent.BlacklistReason reason, Optional<Duration> duration) {
-        logger.trace(hostInfo.getIp(), "IP Blacklisted", reason);
+    private void blacklist(BlacklistHostInfo hostInfo, PeersBlacklistedEvent.BlacklistReason reason) {
+        var ip = hostInfo.getIp();
+        var existingReasonCount = blacklistTracker.findExistingReasonCount(ip, reason);
+        var duration = blacklistTracker.findConfiguredDuration(reason, existingReasonCount);
+
+        logger.trace(format("IP Blacklisted: %s, reason=%s, duration=%s ms, existing count=%s",
```

Comment:
Not sure about the SLF4j version we use allows it, but we should use logger.atTrace().log(...) if possible

Reply:
Just checked, this line uses the LoggerUtil instance in the same way other `trace` calls are made inside this class, so I think this should be OK.

Reply:
ah ok, I thought we were using regular SLF4j. I'd like to get rid of LoggerUtil since it is a performance killer but that's for the future.